multiset.py

```diff
@@ -1087,7 +1087,7 @@ class FrozenMultiset(BaseMultiset):
     __slots__ = ()
 
     def __hash__(self):
-        return hash(tuple(sorted(self._elements.items())))
+        return sum(hash(item) for item in self._elements.items())
```

Comment:
As commented on the issue:
```suggestion
        return hash(frozenset(self._elements.items()))
```

Reply:
I continue to flinch at unnecessary copying, but your suggestion is actually faster than mine anyway, presumably due to spending less time in interpreted code. Done :+1: